src/pages/Auth/Login.jsx

```diff
@@ -22,15 +42,20 @@ export const Login = () => {
           Sign in
         </h2>
         <div className='mt-12'>
-          <form>
+          <form onSubmit={onSubmitHandler}>
             <div>
               <div className='text-sm font-bold text-gray-700 tracking-wide'>
                 Email Address
```

Comment:
Try not to use div for describing what type of data input field is expecting.
Use label instead. This will help you improve accessibility and bigger touch targets.